webapp/src/dataAccess.ts

```diff
@@ -39,10 +38,11 @@ export async function accessLanguage(
 
   await RepoGit.cloneIfNotExist(project);
   const repoGit = await RepoGit.getRepoGit(project);
+  await repoGit.checkoutBaseAndPull();
```

Comment:
It's not safe to `checkoutBaseAndPull` here as there can be concurrent reads from other requests. `main` has this problem too but its milder there as `checkoutBaseAndPull` is only used during `createPullRequest` and during lazy-init of `RepoGit`.

See also `readProject` below.

Let's create a separate concurrency issue and accept a partial regression meanwhile.